stubs/fpdf2/fpdf/fpdf.pyi

```diff
@@ -33,10 +34,19 @@ class Annotation(NamedTuple):
     y: int
     width: int
     height: int
+    flags: tuple[AnnotationFlag, ...] = ...
     contents: str | None = ...
     link: str | int | None = ...
     alt_text: str | None = ...
     action: Action | None = ...
+    color: int | None = ...
+    modification_time: datetime.datetime | None = ...
+    title: str | None = ...
+    quad_points: Sequence[int] | None = ...
```

Comment:
This is typed as a `tuple` in the source code, any reason to use `Sequence` here?

Reply:
Yeah, I saw some place in the source where they make a list and pass it around